mi/reporting/queries/report.nrl-212.provider_snapshot.sql

```diff
@@ -0,0 +1,20 @@
+SELECT
+    MIN(dimension.provider.provider_name) AS provider_name,
```

Comment:
is the MIN here supposed to just return unique provider names or does it do another job as well?